gqcp/include/QCModel/HF/UHF.hpp

```diff
@@ -345,6 +347,656 @@ class UHF {
     }
 
 
+    /**
+     * Construct a mixed-spin component of the spin-conserved stability matrix A'.
+     * 
+     *  @note The formula for this component is as follows:
+     *      A'_I(sigma)A(sigma)J(sigma_bar)B(sigma_bar) = (A(sigma)I(sigma)|J(sigma_bar)B(sigma_bar)).
+     * 
+     *  @note Sigma_bar = alpha if sigma is beta and vice versa.
+     * 
+     *  @note The name `Spin-conserved`comes from the article "Constraints and stability in Hartree-Fock theory" by Seeger, R. and Pople J.A. (https://doi.org/10.1063/1.434318).
+     * 
+     *  @param rsq_hamiltonian      The second quantized Hamiltonian, which contains the necessary two-electron operators.
```

Comment:
Could you clearly indicate that this Hamiltonian is expressed in the non-orthogonal, 'restricted' spin-orbital basis of the AOs?

Please do so for the previous methods you've implemented as well.

Reply:
Adjusted in all `QCModel::HF` classes.